### DISCUSSION TITLE
Advanced Layout Selection like in F1TV+ but even better

### DISCUSSION THREAD

**Discussion by Turrican76** [f1multiviewer/issue-tracker#20] [Feature requests]
The ability to select grid layouts which automatically arrange the feeds in the selected grid. Combining this with a new selection of feeds, video quality, audio feed and volume via drop down menus, would make setting up your own multi view experience quick and easy.

I tried visualizing it by using the F1TV+ Layouts, but numbering them in the grid so that the user knows which feed is for which frame. All done in Word, so please bear with me...

![image](https://user-images.githubusercontent.com/107264253/173178013-d33e710a-af92-4760-881c-455e38ab41c6.png)

**Comment by loheiman** [comment]
Yes I think a few simple preset layouts which satisfy 90% of users. I think the following layouts are all that's needed:
2x2 grid
3x3 grid
1 + 5 (3x3 grid where 2x2 of that is a single feed)

**Reply by beerse** [reply]
Next to the place you save the layout, there is a button to open C:\Users\account\AppData\Roaming\layouts.f1mv.json (replace 'account' with your account name) . You can open and edit this page to adapt your layout. There is a schema file next to it (layouts.schema.json) that can be used as a description file. I have to admit: opening, reading and adapting these files is not for everyone.

The best thing I imagine is that someone can write a dedicated application that finds your display setup and you can arrange the schreens to your liking and save it to be used by F1MultiViewer.

An other idea is that we all join in and share our 'layouts.f1mv.json' files.
